javascript/builtins/intl/NumberFormat.json

```diff
@@ -709,7 +748,7 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": false
+                  "version_added": "preview"
```

Comment:
@queengooborg . I'm getting errors `Error → Experimental should be set to false for  as the feature is supported in multiple browser engines.`

This is because adding this is seen as adding support for a new browser. But in fact it is kind of "partial support" - it could take ages to really be available. 

Is this correct /should I make these no longer experimental?

Reply:
Oops, this will be fixed in #17036!

Reply:
Cool. Rebased.